src/ILCompiler.MetadataTransform/src/ILCompiler/Metadata/IMetadataPolicy.cs

```diff
@@ -43,5 +43,13 @@ public interface IMetadataPolicy
         /// blocked types are dropped from metadata.
         /// </summary>
         bool IsBlocked(Cts.MetadataType typeDef);
+
+        /// <summary>
+        /// Return the Module that should be treated as defining the type. Typically implementations
+        /// will return typeDef.Module, but in some circumstances it may return a different value.
+        /// The ModuleDesc that is returned will be cast to IAssemblyDesc and the GetName method is
+        /// the only function that shall be used on this ModuleDesc.
```

Comment:
```csharp
/// The ModuleDesc that is returned will be cast to IAssemblyDesc and the GetName method is
/// the only function that shall be used on this ModuleDesc.
```

These two lines feel like an unnecessary implementation detail that might become obsolete when we start emitting metadata for modules (and potentially, non-assembly modules).